microsoftbotframework/conversationoperations.py

```diff
@@ -11,7 +11,7 @@ def send(self):
                                     self.conversation['id'],
                                     self.activityId))
 
-        response = self._request(response_url, 'post', self.to_dict())
+        response = self._request(response_url, 'post', self.to_dict(), self.timeout_seconds)
```

Comment:
Please remove the timeout_seconds argument, it must be passed in at class instantiation per your other additions